service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/CloneAllFoldersStep.java

```diff
@@ -94,9 +96,7 @@ public StepResult doStep(FlightContext context) throws InterruptedException, Ret
   @Override
   public StepResult undoStep(FlightContext flightContext) throws InterruptedException {
     var destinationWorkspaceId =
-        flightContext
-            .getInputParameters()
-            .get(ControlledResourceKeys.DESTINATION_WORKSPACE_ID, UUID.class);
+        FlightUtils.getRequired(flightContext.getInputParameters(), WORKSPACE_ID, UUID.class);
```

Comment:
opt nit It's a bit strange that `doStep()` gets destinationWorkspaceId differently. Can you change `doStep()` to this?